osu.Game/Overlays/MusicController.cs

```diff
@@ -237,6 +237,18 @@ protected override void Update()
 
                 if (current.Track.HasCompleted && !current.Track.Looping) next();
             }
+
+            if (title.Size.X > Size.X)

```

Comment:
you'll want to setup a transform and loop it rather than do this in the update loop.